cmd/notary-server/config.json

```diff
@@ -3,10 +3,7 @@
 		"addr": ":4443"
 	},
 	"trust_service": {
-		"type": "remote",
```

Comment:
dev should still run the end to end services. There's not much difference functionally if the server is storing timestamp keys in memory, or if the signer is.